### PR TITLE
Docs/private network installation tutorial

### LINKED ISSUE #268: Include "How to setup an EOSIO Private Network" section in English and Spanish

### PR DESCRIPTION
**Pull request by JustinCast**
### GH Issue

closes #268

### Steps to test
1. `yarn start`
1. Go to `localhost:3000` and search private-network-tutorial resource

#### CheckList
- [x] Follow proper Markdown format
- [x] The content is adequate
- [x] The content is available in both english and spanish
- [x] I Ran a spell check

### CODE REVIEW

File: docs/tutorials/private-network-installation-tutorial.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+- [Seed node: configuration and execution](#seed-node-configuration-and-execution)
+  - [`config.ini`](#configini-2)
+  - [`genesis.json`](#genesisjson-2)
+  - [`start.sh`](#startsh-2)
+- [Stop the nodeos service](#stop-the-nodeos-service)
+- [Reboot the nodeos service](#reboot-the-nodeos-service)
+- [Reboot nodeos from scratch](#reboot-nodeos-from-scratch)
+- [Interacting with the network](#interacting-with-the-network)
+  - [Get stored data](#get-stored-data)
 # Private network installation tutorial
```
Comment (leisterfrancisco) [inline docs/tutorials/private-network-installation-tutorial.md]: Capital case

File: docs/tutorials/private-network-installation-tutorial.md [hunk 2]
```diff
@@ -370,7 +397,7 @@ $ ./start.sh
 ```
 Once the necessary steps to start up the producer nodes it is time to set produder1, producer2 and producer3 accounts as a block producers in the **schedule**, to do so, it is necessary to execute the following command (remember to replace EOS_PUB_DEV_KEY with its respective value):
 ```bash
-$ cleos -u http://localhost:8888 push action eosio setprods {"schedule":[{"producer_name":"productor1","authority": [{"threshold":1,"keys":[{"key":"EOS_PUB_DEV_KEY","weight":1}]}]},{"producer_name":"productor2","authority": [{"threshold":1,"keys":[{"key":"EOS_PUB_DEV_KEY","weight":1}]}]},{"producer_name":"productor3","authority": [{"threshold":1,"keys":[{"key":"EOS_PUB_DEV_KEY","weight":1}]}]}]}
+$ cleos -u http://localhost:8888 push action eosio setprods {"schedule":[{"producer_name":"producer1","authority": [{"threshold":1,"keys":[{"key":"EOS_PUB_DEV_KEY","weight":1}]}]},{"producer_name":"producer2","authority": [{"threshold":1,"keys":[{"key":"EOS_PUB_DEV_KEY","weight":1}]}]},{"producer_name":"producer3","authority": [{"threshold":1,"keys":[{"key":"EOS_PUB_DEV_KEY","weight":1}]}]}]}
 ```
 # API node: configuration and execution
```
Comment (leisterfrancisco) [inline docs/tutorials/private-network-installation-tutorial.md]: Capital case

File: docs/tutorials/private-network-installation-tutorial.md [hunk 3]
```diff
@@ -383,3 +410,379 @@ $ touch start.sh
...
+  tail -f /dev/null & wait ${!}
+done
+
+```
+To start the seed node, simply assign execution permissions to the start.sh file and execute it:
+```bash
+$ chmod 755 start.sh
+$ ./start.sh
+```
+# Stop the nodeos service
```
Comment (leisterfrancisco) [inline docs/tutorials/private-network-installation-tutorial.md]: Capital case

File: docs/tutorials/private-network-installation-tutorial.md [hunk 4]
```diff
@@ -383,3 +410,379 @@ $ touch start.sh
...
+    done
+    echo -ne "\rNode Stopped. \n"
+fi
+```
+Now you can execute `stop.sh` file:
+```bash
+$ chmod 755 stop.sh 
+$ ./stop.sh
+```
+# Reboot the nodeos service
```
Comment (leisterfrancisco) [inline docs/tutorials/private-network-installation-tutorial.md]: Capital case

File: docs/tutorials/private-network-installation-tutorial.md [hunk 5]
```diff
@@ -383,3 +410,379 @@ $ touch start.sh
...
+--p2p-peer-address localhost:9011 \
+--p2p-peer-address localhost:9012 \
+--p2p-peer-address localhost:9013 \
+--hard-replay-blockchain \
+>> $DATADIR"/nodeos.log" 2>&1 & \
+echo $! > $DATADIR"/eosd.pid"
+```
+> **“perhaps we need to replay”**: this error may occur when restarting nodeos due to a missing `--hard-replay-blockchain` flag (parameter that replays all transactions from the genesis node). To fix this error, run the `hard_replay.sh` script.
+
+# Reboot nodeos from scratch
```
Comment (leisterfrancisco) [inline docs/tutorials/private-network-installation-tutorial.md]: Capital case

File: docs/tutorials/private-network-installation-tutorial.md [hunk 6]
```diff
@@ -383,3 +410,379 @@ $ touch start.sh
...
+ls -al
+```
+If you want to delete the current configuration, blockchain data and log files, execute the following commands in the order they appear:
+```bash
+$ cd ~/biosboot/genesis/
+$ ./stop.sh
+$ ./clean.sh
+$ ./genesis_start.sh
+```
+# Interacting with the network
```
Comment (leisterfrancisco) [inline docs/tutorials/private-network-installation-tutorial.md]: Capital case

File: docs/tutorials/private-network-installation-tutorial.md [hunk 7]
```diff
@@ -383,3 +410,379 @@ $ touch start.sh
...
+At this point, the network is able to host any contract. Let's suppose you want to deploy a contract like [easycontract](https://github.com/eoscostarica/eoscr-smart-contracts/tree/master/easycontract) EOS Costa Rica's example repo. After you follow the specified instructions there, you can call easycontract's `save` ACTION like following:
+```bash
+cleos -u NODE_URL_HERE push action easycontract save '{"date":"2021/01/10 21:01:35","filename":"ejemplo.pdf","path":"/var/www/docs"}' -p easycontract@active
+```
+After the execution, you will get an output in the terminal.
+## Get stored data
```
Comment (leisterfrancisco) [inline docs/tutorials/private-network-installation-tutorial.md]: Capital case

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: docs/tutorials/private-network-installation.md [hunk 8]
```diff
@@ -30,7 +37,7 @@
 # Private network installation tutorial
 Several topologies can be designed in the sense of quantity of nodes or redundancy of the data, nevertheless, for the practicity of this tutorial, the following topology is used as a main reference:
 <p style={{ align: "center" }}>
-  <img src={useBaseUrl( '/img/private-network-installation-tutorial/initial-topology.png' )} width="100%" />
+  <img src={ useBaseUrl( '/img/private-network-installation-tutorial/initial-topology.png' )} width="100%" />
 </p>
 
 # Software requirements
```
Comment (leisterfrancisco) [inline docs/tutorials/private-network-installation.md:43]: Title Case

File: docs/tutorials/private-network-installation.md [hunk 9]
```diff
@@ -30,7 +37,7 @@
 # Private network installation tutorial
```
Comment (leisterfrancisco) [inline docs/tutorials/private-network-installation.md:37]: Title Case

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by leisterfrancisco — DISMISSED** [review]
Good work 👍

**Review by Edgar097 — DISMISSED** [review]

File: docs/tutorials/private-network-installation.md [hunk 10]
```diff
@@ -0,0 +1,794 @@
...
+- [Seed Node: Configuration and Execution](#seed-node-configuration-and-execution)
+  - [`config.ini`](#configini-2)
+  - [`genesis.json`](#genesisjson-2)
+  - [`start.sh`](#startsh-2)
+- [Stop the nodeos Service](#stop-the-nodeos-service)
+- [Reboot the nodeos Service](#reboot-the-nodeos-service)
+- [Reboot nodeos from Scratch](#reboot-nodeos-from-scratch)
+- [Interacting with the Network](#interacting-with-the-network)
+  - [Get Stored Data](#get-stored-data)
+# Private Network Installation Tutorial
```
Comment (xavier506) [inline docs/tutorials/private-network-installation.md:37]: Lets start with H2 tags since the H1 is always the page title

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**